aldryn_config.py

```diff
@@ -17,6 +17,7 @@ class Form(forms.BaseForm):
         'Enable permission checks',
         required=False,
         initial=True,
+        help_text='When set, provides new fields in each page's settings to assign levels of access to particular users.',
```

Comment:
You'll have to mix quotes - use `"` for quting the string, then you can leave `page's` as is.